Connectors/Autocad/Speckle.Connectors.AutocadShared/HostApp/AutocadInstanceObjectManager.cs

```diff
@@ -16,17 +16,19 @@ namespace Speckle.Connectors.Autocad.HostApp;
 /// <inheritdoc/>
 ///  Expects to be a scoped dependency per send or receive operation.
 /// </summary>
-public class AutocadInstanceObjectManager : IInstanceObjectsManager<AutocadRootObject, List<Entity>>
+public class AutocadInstanceObjectManager : IInstanceUnpacker<AutocadRootObject>, IInstanceBaker<List<Entity>>
```

Comment:
give `AutocadInstanceObjectManager` it's own interface that inherits from the new ones that use that for injection

Reply:
Regardless, we will split this class into 2 pieces as baker and unpacker. I already put a POC comment about it, probably we will do it as you said when we do it. We want to merge this PR in asap for perf tests by Bilal and from us.

Reply:
creating a followup ticket for this